tools/accuracy_checker/accuracy_checker/adapters/classification.py

```diff
@@ -73,15 +81,20 @@ def process(self, raw, identifiers, frame_meta):
         if self.block:
             if self.argmax_output:
                 single_prediction = ArgMaxClassificationPrediction(identifiers[0], prediction)
+            elif self.fixed_output:
+                single_prediction = ArgMaxClassificationPrediction(identifiers[0],
```

Comment:
does it mean that batching is not supported for this model type?

Reply:
No, model supports batching. For each batch model outputs pair of values, direct and reverse probability, we need only direct one.